Book - Update logistics

jomey opened the pull request:
Remove SnowEx wording and update email contact.

Where is the event again?

Thought it would be good to have some information in place when sending out the first round of acceptance letters.

Also, the *Dorm* and *Meals* information needs to be verified.

jomey committed:
Book - Update logistics

Remove SnowEx wording and update email contact.

jomey committed:
Logistics - Add link to Dorms

scottyhq commented on book/logistics.md:
```diff
@@ -8,30 +8,55 @@
-* Doors open at 8:30 am each morning (Monday-Friday). We will start the morning sessions at 9:00 am sharp.
-* Due to space limitations, only registered SnowEx participants will be allowed to attend the tutorials and open working sessions.
-* Participants are expected to attend all scheduled SnowEx activities. Contact uwhackweek@uw.edu if any circumstances prevent you from attending.
+* Tutorials and courses will occur in the _REPLACE_.
```
```suggestion
* Tutorials and courses will occur in the [University of Washington South Campus Center](https://hsasf.hsa.washington.edu/instructional-support/crs/matrix/scc/) Room 221.
```

scottyhq replied:
https://github.com/orgs/ICESAT-2HackWeek/projects/10/views/2?pane=issue&itemId=29496365

scottyhq commented on book/logistics.md:
```diff
@@ -8,30 +8,55 @@
+  can store your luggage if your flight leaves later that day. Please let us
+  know immediately by emailing icesat2-hackweek@uw.edu if you plan on arriving/departing
+  at an earlier/later date.
+* The front desk can direct you to the _REPLACE_ where the meetings
```
```suggestion
* The front desk can direct you to the South Campus Center where the meetings
```

jomey committed:
Logistics - Add event location

Co-authored-by: Scott Henderson <scottyhq@gmail.com>

jomey committed:
Logistics - Add event location - Part II

Co-authored-by: Scott Henderson <scottyhq@gmail.com>

jomey committed:
QAQC - Switch to use `.codespellignore` file for ignored words.

Make it easier to add words to the ignore file. This should also fix the
PR for this branch.

jomey commented on .github/workflows/qaqc.yaml:
```diff
@@ -33,7 +33,7 @@ jobs:
         check_hidden: true
         skip: '*.js,qaqc.yml'
-        ignore_words_list: slippy,trough,thw
+        ignore_words_file: .codespellignore
```
@scottyhq - Note the change to use a file instead of a list in the action. Should make it easier for tutorial leads to edit and add words.
